### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by yujhongmicrosoft**
Edited file names

### CODE REVIEW

**Comment by yujhongmicrosoft** [comment]
#sign-off

File: README.md [hunk 1]
```diff
@@ -104,11 +104,12 @@ These need to be further massaged in MongoDB.
...
-        |role-Xx.json|-c role|
-        |policy-Xx.json|-c policy|
-        |portalextension-Xx.json|-c portal**E**xtension|
-        
+        |roles-Xx.json|-c role|
+        |policies-Xx.json|-c policy|
+        |portalextensions-Xx.json|-c portal**E**xtension|
+            
     * Files need to be loaded in the following order: Ww, Ff, Mc, Bf. Note that all the files for a given type, irrespective of the cloud, are loaded into the same collection.
+    * When importing **portalextensions**, add ```--jsonArray ``` at end of import command.
```
Comment (gsacavdm) [inline README.md:112]: This shouldn't be necessary. The content of the portal extensions should be all contained within a `value` property like so:

```json
{
  "value" : [
    
    {
      "name": "HubsExtension",
      "validEndpoint": false,
      "regexMatch": false
    },
    {
      "name": "Microsoft_AAD_B2CAdmin",
      "features": {},
      "validEndpoint": true,
      "regexMatch": true
    },
    ...
  ]
}
```

See [Program.cs#L262](https://github.com/gsacavdm/azure-parity/blob/master/Program.cs#L262)

**Comment by gsacavdm** [comment]
File names updates look good but the comment on adding `--jsonArray` shouldn't be necessary.
Remove that and I'll merge this.

Let's chat separately about that other change, it shouldn't be necessary as per my comment.

**Comment by gsacavdm** [comment]
Have made changes so that this is no longer necessary,

**Pull request closed without merging**